manifests/Adobe/AdobeAcrobatReaderDC/2020.009.20063.yaml

```diff
@@ -0,0 +1,15 @@
+Id: Adobe.AdobeAcrobatReaderDC
+Version: 2020.009.20063
+Name: Adobe Acrobat Reader DC
+Publisher: Adobe
+License: Copyright (c) Adobe
+LicenseUrl: https://www.adobe.com/content/dam/cc/en/legal/licenses-terms/pdf/Reader-EULA-en_US-20181207.pdf
+AppMoniker: adobereader
+Description: Only with Adobe Acrobat Reader you can view, sign, collect and track feedback, and share PDFs for free. 
```

Comment:
```suggestion
Description: The free global standard for reliably viewing, printing, signing, and commenting on PDF documents.
```

Reply:
Jesus which one should I choose

Reply:
@Reset12138 I assume this is the most appropriate description given by Adobe. See this screenshot:
![image](https://user-images.githubusercontent.com/45366162/82647977-49873500-9c17-11ea-9507-0e3c53ec5a55.png)

Reply:
> @Reset12138 I assume this is the most appropriate description given by Adobe. See this screenshot:
> ![image](https://user-images.githubusercontent.com/45366162/82647977-49873500-9c17-11ea-9507-0e3c53ec5a55.png)

got it.
already merge yours